### ISSUE TITLE
Bit requires older Node version on Mac

### ISSUE THREAD

**Issue by runnwerth** [teambit/bit#5807]
### Description
When running bit (in this case specifically "bit templates") there is the following error:

Node version 18.0.0 is not supported, please use Node.js >=12.22.0 <17.0.0.
If you must use legacy versions of Node.js, please use our binary installation methods. https://docs.bit.dev/docs/installation

### Specifications
* Bit version: 0.0.732 (per download)
* Node version: v18.0.0
* npm / yarn version: 8.10.0
* Platform: Mac OS Monterey 12.3.1
* Bit compiler (include version): How do I find that out?
* Bit tester (include version): How do I find that out?

### Context and additional information

Getting started with bit on a MAC. Everything worked well on Ubuntu.

**Comment by itaymendel** [comment]
will be resolved when this is merged - https://github.com/teambit/bvm/pull/44
tl;dr - bit will not depend on the node env from your system.
